config/gocdb_schema.xml

```diff
@@ -604,7 +604,7 @@
         <field>
             <fname>TYPE</fname>
             <length>255</length>
-            <regex>/^(X509|OIDC Subject)$/</regex>
+            <regex>/^(X\.509|OIDC Subject)$/</regex>
```

Comment:
What does this impact?

Reply:
This (and the related changes further "up" in the code) mean that a user would have to edit any existing API authentication entities they still want to use in order for them to pick up the new "X.509" type.

Because of #284, users will have to edit existing API authentication entities following the release of GOCDB 5.10.0 anyway, to associate the entities with a user.